test/over_react/component_declaration/builder_integration_tests/abstract_accessor_integration_test.dart

```diff
@@ -0,0 +1,240 @@
+// Copyright 2016 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import 'package:over_react/over_react.dart';
+import 'package:test/test.dart';
+
+// ignore: uri_has_not_been_generated
+part 'abstract_accessor_integration_test.over_react.g.dart';
```

Comment:
Just occurred to me that with build to source, we won't need the ignore comment here.

Reply:
Did you mean to say "build to cache"?

Reply:
I meant to say build to source – we've been experimenting with it and I was able to confirm this weekend that it works and is backwards-compat with Dart 1. Our hope is that it would provide a perf benefit because it also means we only have to build the root package and not all packages

Reply:
For posterity, we're sticking with build to cache over source.